examples/components/simpleviewer.js

```diff
@@ -88,10 +88,12 @@ const loadingTask = pdfjsLib.getDocument({
   cMapPacked: CMAP_PACKED,
   enableXfa: ENABLE_XFA,
 });
-loadingTask.promise.then(function (pdfDocument) {
+
+(async function () {
   // Document loaded, specifying document for the viewer and
   // the (optional) linkService.
+  const pdfDocument = await loadingTask;
```

Comment:
Sorry, but this doesn't actually work!
The `loadingTask` is *not* a Promise, but rather a structure with a `promise`-property; see https://github.com/mozilla/pdf.js/blob/f6d9d91965a97c80b6f5b335a27cdb4d36b1b9e2/src/display/api.js#L527-L601

Hence the *correct* change would be `const pdfDocument = await loadingTask.promise;`.

Furthermore, given that the example cannot possibly work as-is: please make sure to (in this case manually) test the changes that you make to ensure that your code actually works.
Testing this example locally, after making changes, includes:
 - Running `gulp dist-install`.
 - Running `gulp server`.
 - Opening http://localhost:8888/examples/components/simpleviewer.html in a browser and check that the example loads and works correctly without e.g. any errors printed in the console.